graphic-analysis-frontend/src/components/form/FormLogin.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useContext } from "react";
+import { LoginContext } from "../../context/LoginContext";
+import { Stack, TextField, Button, Divider } from "@mui/material";
+import styles from "../../pages/signIn/styles.module.css";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import Router from "next/router";
+import { baseUrl } from "../../config/api";
```

Comment:
Geralmente eu organizo os imports pra facilitar entrar o que procuro. (Coisas de outras libs ou react, coisas que eu implementei, como components e por fim css quando tem)
```suggestion
import Router from "next/router";
import { ToastContainer, toast } from "react-toastify";
import React, { useState, useContext } from "react";
import { Stack, TextField, Button, Divider } from "@mui/material";

import { baseUrl } from "../../config/api";
import { LoginContext } from "../../context/LoginContext";

import "react-toastify/dist/ReactToastify.css";
import styles from "../../pages/signIn/styles.module.css";
```